src/components/Table/GridTable.tsx

```diff
@@ -608,16 +679,29 @@ const VirtualRoot = memoizeOne<
     firstLastColumnWidth: number | undefined,
     xss: any,
   ) => Components["List"]
->((gs, columns, id, firstLastColumnWidth, xss) => {
+>((gs, _columns, id, _firstLastColumnWidth, xss) => {
   return React.forwardRef(function VirtualRoot({ style, children }, ref) {
+    // This VirtualRoot list represent the header when no styles are given. The
+    // table list generally has styles to scroll the page for windowing.
+    const isList = Object.keys(style || {}).length !== 0;
+
     // This re-renders each time we have new children in the view port
     return (
       <div
         ref={ref}
+        /** Styles from React-Virtuoso. You should probably not be touching this... */
         style={style}
         css={{
           // Add an extra `> div` due to Item + itemContent both having divs
           ...Css.addIn("& > div + div > div > *", gs.betweenRowsCss || {}).$,
+          // Add `display:contents` to Item to flatten it like we do GridRow
+          ...Css.addIn("& > div", Css.display("contents").$).$,
```

Comment:
Hasn't `display:contents` gone away for virtualized tables @bdow ? Fwiw this might have _just_ landed, but in theory we wouldn't want this?

Reply:
Yes, this is accurate. Chatting with Kolton now about it.

Reply:
Great! This should be removed now